boltdesignsystem/bolt
# DS-867 (Part 2 of 2) Restructure image folder, update docs pages

## Jira

https://pegadigitalit.atlassian.net/browse/DS-867

## Summary

* Refactored the Doc Site image folder and updated the documentation examples to reflect theses changes.

## Details

### Restructured the Doc Site image folder
- Removed `images/backgrounds/`
- Removed `images/content/backgrounds/`
- Created `images/content/academy/`
- Created `images/content/best-of-content/`
- Created `images/content/calculator/`
- Created `images/content/contact/`
- Created `images/content/heroes/`
- Created `images/content/icons/`
- Created `images/content/layout/`
- Removed `images/content/logos/480x160`
- Created `images/content/oclp/`
- Removed `images/content/portraits`
- Created `images/content/profiles/`
- Removed `images/content/promos`
- Removed `images/content/screenshots`
- Created `images/content/user/`
- Removed `images/docs`
- Removed `images/heros`
- Removed `images/logos`
- Created `images/placeholders/backgrounds/`
- Removed `images/shapes`

Moving forward, we should be utilizing the couple options that are in the `images/placeholders/` folder for most Doc Site images. There should be enough images variation to support most of what we need for test images. We can modify or add more moving forward.

The content folder should be utilized when we need to build Drupal specific mockups that requires that will be used to build out a design. These images are just for display purposes and should not be shipped to Drupal (Drupal has its own image management system).

### Updated the Documentation pages
* Removed all unused images from the Doc Site image folder.
* Updated existing images examples on the Doc Site to the placeholder images examples (if applicable).
* Fixed the broken images on the archived Academy pages.

## How to test

- Pull down the branch and test the `yarn start`, `yarn build` and `yarn tests:js` commands.
- Click though the doc site to make sure no images are broken (you can quickly see this in a browsers dev tools). 
-- Note here, I did a very careful review of all the pages so I am fairly certain there are no broken images (no promises though) so i think we would be fine with just focusing on components/elements/pages that we know have a lot of images. For example, review the Background Component, Image Element and Academy pages among others.

## Release notes

- Refactored the Doc Site image folder
- Updated the Doc Site pages images to new image folder paths
